src/components/container/PartnerBanner/PartnerBanner.scss

```diff
@@ -0,0 +1,9 @@
+.PartnerBanner {
+    background-color: var(--color-bunting);
+    
+    img {
+        height: 2em;
+        object-fit: contain;
+        margin: 1.25em;
+    }

```

Comment:
we might change the sizes later on - to match the design, like 
 img {
        height: 1.1em;
        object-fit: contain;
        margin: 1.4em;
    }